src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
exctra empty line and make ColorSuplier class field. If you have a questions about this, send a message in you chanal)